dll/win32/themeui/lang/cs-CZ.rc

```diff
@@ -1,10 +1,18 @@
+/*
+ * PROJECT:     ReactOS Theme UI Library
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Czech resource file
+ * TRANSLATORS: Copyright 2014 Radek Liška (Black_Fox) <radekliska@gmail.com>
```

Comment:
```suggestion
 * TRANSLATORS: Copyright 2018 Lee Schroeder <spaceseel@gmail.com>
```
Ref. 94e4c696e559

Reply:
@binarymaster Radek Liška translated the names of styles in f3630ec, in 94e4c69 they were copied into separate resource file.

Reply:
Ah ok.